py/selenium/webdriver/common/options.py

```diff
@@ -87,3 +87,33 @@ def to_capabilities(self):
     @property
     def default_capabilities(self):
         return {}
+
+class PrintOptions():
```

Comment:
A class should guide a user on how to use it via the APIs it offers.

We should add setter properties for each of the values that people want to set. At the moment we don't validate that people are setting the right thing